rasa_nlu/data_router.py

```diff
@@ -151,23 +151,47 @@ def _collect_projects(self, project_dir):
         projects.extend(self._list_projects_in_cloud())
         return projects
 
-    def _create_project_store(self, project_dir):
+    def _create_project_store(self,
+                              project_dir):
         projects = self._collect_projects(project_dir)
 
         project_store = {}
 
         for project in projects:
-            project_store[project] = Project(self.component_builder,
-                                             project,
-                                             self.project_dir,
-                                             self.remote_storage)
+            if self.model_server is not None:
+                project_store[project] = load_from_server(
```

Comment:
I wanted to make the remote models available in all projects. But I agree it's a bit strange. How about a fixed name for the project that loads from remote, e.g. `remote` or just `default`?

Reply:
those this makes sense though for multiple projects? because it is pulling the same model in all projects, right?

Reply:
no in that case we wouldn't loop over projects. it would be something like
```python
if self.model_server is not None:
    project_store["remote"] = load_from_server()
else:
   for project in project_store:
        (...)
```
does that make sense?

Reply:
yes I know what you mean. I was just thinking if it does make sense to do it for all projects, but I guess not as long as we only have one url, right?

Reply:
it doesn't really, but we could implement it later if we want to support multiple pull URLs. in that case the `EndpointConfig` could be easily configured for that.
For now, shall we stick to the `default` project name for remote pulling?

Reply:
yes I think that makes sense. needs to be explained in the docs :+1: